contract_mandate/__manifest__.py

```diff
@@ -4,7 +4,7 @@
 {
     "name": "Contract Mandate",
     "summary": "Mandate in contracts and their invoices",
-    "version": "14.0.1.0.0",
+    "version": "16.0.1.0.0",
     "author": "Odoo Community Association (OCA), " "Tecnativa",
```

Comment:
Hi, can you fix the typo created by an old pre-commit ?

```suggestion
    "author": "Odoo Community Association (OCA), Tecnativa",
```